.github/scripts/resolve_plugins.py

```
@@ -0,0 +1,45 @@
+        ref = ver[1:]
+        git_url = pkg.removeprefix("git+")
+        try:
+            out = subprocess.run(  # noqa: S603
+                ["git", "ls-remote", git_url, ref],  # noqa: S607
+                check=False,
+                capture_output=True,
+                text=True,
+                timeout=15,
+            )
+            sha = out.stdout.split()[0] if out.stdout else ref
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)